.circleci/workflows.yml

```
@@ -2739,6 +2763,25 @@ linux-x64-workflow: &linux-x64-workflow
           - run-reporter-component-tests-chrome
           - run-webpack-dev-server-integration-tests
           - run-vite-dev-server-integration-tests
+    # Cypress run must be completed to fetch Accessibility report
+    - verify-accessibility-results:
+        context: test-runner:cypress-record-key
+        requires:
```

are these checks able to complete sooner so we can fail faster if violations are detected?

---

@AtofStryker No, the Cypress run has to complete before the report is generated so moving this up results in unnecessary polling.